tedana/selection/_utils.py

```diff
@@ -8,6 +8,17 @@
 LGR = logging.getLogger(__name__)
 
 
+def reorder_dataframe(comptable):
+    """
+    Reorder columns in component table and clean up rationale column.
```

Comment:
Could you elaborate slightly more for this docstring? It looks from the code that you're keeping only the classification and rationale columns, while stripping out semicolons from the rationale, same as the original larger module. It may seem straightforward but I think it's still helpful to note this in the docstring. It's true that this level of comment detail was absent in the original, but with this as an independent function I think it would be very useful.